tensorboard_plugin/tensorboard_plugin_fairness_indicators/plugin_test.py

```diff
@@ -150,6 +151,12 @@ def testIndexJsRoute(self):
     response = self._server.get("/data/plugin/fairness_indicators/index.js")
     self.assertEqual(200, response.status_code)
 
+  @pytest.mark.xfail(
+    reason=(
+      "This test is currently failing. "
+      "Once it is fixed, please remove this mark."
```

Comment:
```suggestion
    "Failing on `master` as of 942b672457e07ac2ac27de0bcc45a4c80276785c. Please remove once fixed."
```

Can you also make an issue for this?

Reply:
Made the change and the issue (#388). Also reverted the formatting changes.